app/src/main/java/com/framgia/fsalon/screen/report/customerreport/CustomerReportViewModel.java

```diff
@@ -0,0 +1,161 @@
+package com.framgia.fsalon.screen.report.customerreport;
+
+import android.content.Context;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.support.v4.app.Fragment;
+import android.support.v4.content.ContextCompat;
```

Comment:
ERROR: Unused import: android.support.v4.content.ContextCompat. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports